### PR TITLE
Doc: pipenv shell --fancy

### PR DESCRIPTION
**Pull request by tapaswenipathak**
### The issue

Fixes https://github.com/pypa/pipenv/issues/3491.

### The fix

Just added a tiny line exposing the potential issues.

### The checklist

* [X] Associated issue
* [X] A news fragment in the `news/` directory to describe this fix with the extension `.bugfix`, `.feature`, `.behavior`, `.doc`. `.vendor`. or `.trivial` (this will appear in the release changelog). Use semantic line breaks and name the file after the issue number or the PR #.

### CODE REVIEW

File: pipenv/cli/command.py [hunk 1]
```diff
@@ -342,7 +342,8 @@ def lock(
     "--fancy",
     is_flag=True,
     default=False,
-    help="Run in shell in fancy mode (for elegantly configured shells).",
+    help="Run in shell in fancy mode. Make sure the shell have no path manipulating
```
Comment (frostming) [inline pipenv/cli/command.py]:
```suggestion
    help="Run in shell in fancy mode. Make sure the shell have no path manipulating"
```
No line breaking in a quoted string, but you can use implicit string concatenation

File: pipenv/cli/command.py [hunk 2]
```diff
@@ -342,7 +342,8 @@ def lock(
     "--fancy",
     is_flag=True,
     default=False,
-    help="Run in shell in fancy mode (for elegantly configured shells).",
+    help="Run in shell in fancy mode. Make sure the shell have no path manipulating"
+          "scripts. Run $pipenv shell for issues with compatibility mode.",
```
Comment (frostming) [inline pipenv/cli/command.py]: just a minor tweak: can you align the two beginning quotes?

**Comment by tapaswenipathak** [comment]
@frostming: lemme know now.

File: pipenv/cli/command.py [hunk 3]
```diff
@@ -342,7 +342,8 @@ def lock(
     "--fancy",
     is_flag=True,
     default=False,
-    help="Run in shell in fancy mode (for elegantly configured shells).",
+    help="Run in shell in fancy mode. Make sure the shell have no path manipulating"
+         "scripts. Run $pipenv shell for issues with compatibility mode.",
```
Comment (frostming) [inline pipenv/cli/command.py]: Thanks for your quick action, only a last comment: a whitespace is needed before `scripts`

**Pull request merged**